cypress/fixtures/analysis.sh

```
@@ -0,0 +1,243 @@
+#!/bin/bash
+
+set -e
+
+rawHost="${HOST:-localhost:8080}"
+if [[ "$rawHost" == *"/hub"* ]]; then
+  host="$rawHost"
+else
+  host="${rawHost}/hub"
+fi
+  
+export TOKEN=$(curl -kSs -d '{"user":"admin","password":"Dog8code"}' \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)